libs/blocks/marquee/marquee.css

```diff
@@ -189,10 +216,23 @@
   margin-bottom: var(--spacing-s);
 }
 
+.marquee.split a:not(.con-button) {
+  color: var(--color-accent);
+  text-decoration: underline;
+}
```

Comment:
This seems over reaching. We probably want to rely on site defaults for general link color/underline behavior. If those need to change? we should do it in styles/variables.css globally and make this a separate task.

Reply:
@ryanmparrish  Makes sense to me. I've removed this rule, but this does make the simple links [look incorrect now](https://split-marquee--milo--adobecom.hlx.page/drafts/ebartholomew/split-marquee?martech=off). Should we create the separate task for that styling and address that before continuing with this PR?